big_tests/tests/domain_removal_SUITE.erl

```diff
@@ -1,21 +1,6 @@
 -module(domain_removal_SUITE).
 
-%% API
--export([all/0,
-         groups/0,
-         init_per_suite/1,
-         end_per_suite/1,
-         init_per_group/2,
-         end_per_group/2,
-         init_per_testcase/2,
-         end_per_testcase/2]).
-
--export([mam_pm_removal/1,
-         mam_muc_removal/1,
-         inbox_removal/1,
-         muc_light_removal/1,
-         muc_light_blocking_removal/1,
-         private_removal/1]).
+-compile(export_all).
```

Comment:
Add please also `-compile(nowarn_export_all).`.

Reply:
Good idea, but we don't use in any test suite, so I think we could do it separately for all of them.

Reply:
I'm try add it in separate PR 🙃